components/user/UserEvents.js

```diff
@@ -1,17 +1,26 @@
 import EventCard from "../event/EventCard";
 import Alert from "../Alert";
 import EventKey from "../event/EventKey";
+import { useState } from "react";
```

Comment:
This doesn't seem to be used?

Reply:
I didn't get you can it be done without useState variables

Reply:
This import line is added to the file but I don't see this import used in the file?